docs/recipes/gulp/gulpfile.js

```diff
@@ -9,18 +9,16 @@
 const gulp = require('gulp');
 const connect = require('gulp-connect');
 const lighthouse = require('lighthouse');
-const ChromeLauncher = require('lighthouse/lighthouse-cli/chrome-launcher').ChromeLauncher;
+const chromeLauncher = require('lighthouse/chrome-launcher');
```

Comment:
Ah ok. Disabling the OctLinker extension puts everything back to normal :) Sorry for the confusion.